tests/integrate_test/cita_features_test.sh

```diff
@@ -55,6 +62,11 @@ main() {
     echo -n "5) Check permission denied ... "
     test_perm_denied 2
     echo "Done"
+
+    echo -e "6) Run block interval tests ...\n"
+    test_change_block_interval 2
+    timeout=$(check_height_growth_normal 0 60) || (echo "Block interval changed")
```

Comment:
the block_intercal is `1000000000`, How `check height growth normal`?

Reply:
This check will failed for the block interval larger moren than 60 and exec echo operation.

Reply:
then how to check and ensure it?  Can it(the test case) fail if `height growth normal`?

Reply:
Check it again